global/scripts/processingmodecheck.shlib

```diff
@@ -0,0 +1,54 @@
+#!/bin/echo This script should not run directly:
+
+# Functions for checking compliance with selected MPP mode for HCP Pipelines
+
+# --- Import log.shlib 
+
+if [[ "$(type -t log_Err_Abort)" != "function" ]]
+then
+    source "$HCPPIPEDIR/global/scripts/log.shlib" "$@"
+fi
+
+
+check_mode_compliance()
+{
+
+    # --- Define variables
+
+    local ProcessingMode="$1"
+    local Compliance="$2"
+    local ComplianceMsg="$3"
+
+    # --- Initial information
+
+    if [ "${ProcessingMode}" = "LegacyStyleData" ]; then
+      log_Msg "LegacyStyleData mode requested."
+      log_Msg "NOTICE: You are using a mode that enables processing of acquisitions that do not"
+      log_Msg "        conform to the HCP specification as described in Glasser et al. (2013)!"
+      log_Msg "        Be aware that if the HCP requirements are not met, the level of data quality"
+      log_Msg "        can not be guaranteed and the Glasser et al. (2013) paper should not be used"
+      log_Msg "        in support of this workflow. A manuscript with comprehensive evaluation for"
+      log_Msg "        the Extended MPP workflow is in active preparation and should be appropriately"
```

Comment:
We should consider removing "MPP" here too, perhaps say "legacy data workflow" instead?

Reply:
Absolutely!  Don't want to let "MPP" slip in given all the work we've done to avoid it.

Reply:
It's like weeds ... so difficult to root out!